BioForm/profiles/management/commands/setuserinactive.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management.base import BaseCommand
+from profiles.models import Account
+from datetime import datetime, timedelta
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        days_offline = 10
+
+        inactive_users = Account.objects.filter(last_login__lte=datetime.now()-timedelta(days=days_offline))
+
+        if inactive_users:
+            for user in inactive_users:
+                
+                count += 1
+                user.is_active = False
+
+                user.save()
+        
+        print(f"{count} user(s) were set to inactive")
```

Comment:
You can update all users in one database query, database queries are expensive it's better to minimize them where ever ossible. [Reference](https://stackoverflow.com/questions/2712682/how-to-select-a-record-and-update-it-with-a-single-queryset-in-django)
```suggestion
        inactive_users_queryset = Account.objects.filter(last_login__lte=datetime.now()-timedelta(days=days_offline))
        inactive_users_count = inactive_users_queryset.count()
        inactive_users_queryset.update(is_active=False)
        
        print(f"{inactive_users_count} user(s) were set to inactive")
```